spec/models/program_spec.rb

```diff
@@ -199,4 +199,21 @@
     end
   end
 
+  describe '#any_event_for_this_date?' do
+    it 'returns type should be boolean' do
+      expect(program.any_event_for_this_date?('').in?([true, false])).to eq(true)
```

Comment:
I don't know what it is that makes rubocop fail. You need to show me the code and the failure.